### PR TITLE
[3.9] en-GB.plg_captcha_recaptcha.ini: Alpha sorting strings. Taking off blank lines before comments.

### PR DESCRIPTION
**Pull request by infograf768**
As title says.

Blank lines prevent seeing the comments on some translation systems.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.plg_captcha_recaptcha.ini [hunk 1]
```diff
@@ -3,73 +3,70 @@
...
+PLG_RECAPTCHA_ERROR_INVALID_SITE_PRIVATE_KEY="We weren't able to verify the secret key."
+PLG_RECAPTCHA_ERROR_INVALID_SITE_PUBLIC_KEY="We weren't able to verify the site key."
+PLG_RECAPTCHA_ERROR_NO_IP="For security reasons, you must pass the remote IP address to reCAPTCHA."
+PLG_RECAPTCHA_ERROR_NO_PRIVATE_KEY="reCAPTCHA plugin needs a secret key to be set in its parameters. Please contact a site administrator."
+PLG_RECAPTCHA_ERROR_NO_PUBLIC_KEY="reCAPTCHA plugin needs a site key to be set in its parameters. Please contact a site administrator."
 PLG_RECAPTCHA_ERROR_RECAPTCHA_NOT_REACHABLE="Unable to contact the reCAPTCHA verify server."
-
+PLG_RECAPTCHA_ERROR_UNKNOWN="Unknown error."
+PLG_RECAPTCHA_ERROR_VERIFY_PARAMS_INCORRECT="The parameters to verify were incorrect, make sure you are passing all the required parameters."
 PLG_RECAPTCHA_PRIVACY_CAPABILITY_IP_ADDRESS="The reCAPTCHA plugin integrates with Google's reCAPTCHA system as a spam protection service. As part of this service, the IP of the user who is answering the captcha challenge is transmitted to Google."
```
Comment (zero-24) [inline administrator/language/en-GB/en-GB.plg_captcha_recaptcha.ini]: Should't we add before this line ";Privacy notice" as this is no error message at all ;)

Comment (infograf768) [reply administrator/language/en-GB/en-GB.plg_captcha_recaptcha.ini]: done

**Comment by zero-24** [comment]
Looks good to me. Lets give Brian a bit more time to take a look so we can merge here. 👍

**Review by brianteeman — APPROVED** [review]

**Comment by brianteeman** [comment]
if blank lines are a problem then there are more files to review and fix

**Comment by zero-24** [comment]
Ok merging this for now and the other one can be patched later. Thanks.

**Comment by infograf768** [comment]
> if blank lines are a problem then there are more files to review and fix

I did that one because it was really a mess and time to fix its alpha order already in 3.9.

For the others, it is better to postpone to 4.0 imho.

**Comment by brianteeman** [comment]
If it's a real problem, and I trust you when you say it is, then it should be fixed in j3 as that has a minimum of two years support when people need to translate joomla.

**Pull request merged**